.cargo/config.toml

```diff
@@ -12,8 +12,12 @@ rustflags = [
   "link-arg=/STACK:8000000",
 ]
 
-# Required for `dist` to work with linux arm targets: https://github.com/axodotdev/cargo-dist/issues/74#issuecomment-2053680080
 [env]
+PIXI_CONFIG_DIR = "pixi"
+PIXI_DEFAULT_CHANNELS = "conda-forge"
+PIXI_DIR = ".pixi"
+PIXI_PROJECT_LOCK_FILE = "pixi.lock"
```

Comment:
This makes sense for `magic`, but since we use the defaults we don't need to override anything.
```suggestion
```

Reply:
Ack. This was an example for how to do it. I'll remove.